.travis.yml

```diff
@@ -11,6 +11,8 @@ matrix:
         - os: linux
           python: 3.5
         - os: linux
+          python: 3.6
+        - os: linux
```

Comment:
(I would prefer that to be in a separate PR, since I've zero practical experience with macs or homebrew)